week3/assignment/src/components/ByType/ByTypeResult.jsx

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import { Question, SectionWrapper } from "../../style/MainStyle";
+import DATA from "../../assets/DATA";
+
+const ByTypeResult = (props) => {
+  const {
+    isStart,
+    setIsStart,
+    isStartHandler,
+    selectedMood,
+    selectedDegree,
+    selectedCnt,
+  } = props;
+
+  const resetState = () => {};
+  const backToFront = () => {};
+  const resetIsStart = () => {
+    isStart ? setIsStart(false) : setIsStart(true);
+  };
+
+  let resIdx = 0;
+  DATA.forEach((elm, idx) => {
+    if (
+      elm.mood === selectedMood &&
+      elm.degree === selectedDegree &&
+      elm.teamOrSolo === selectedCnt
+    ) {
+      resIdx = idx;
+    }
+  });
+  const AlbumCover = DATA[resIdx].img;
+  const SongTitle = DATA[resIdx].title;
+
```

Comment:
```suggestion
let typeImg = DATA.find((ele) => {
ele.mood === selectedMood && ele.degree === selectedDegree && ele.teamOrSolo === selectedCnt
}
const AlbumCover = typeImg.img;
const SongTitle = typeImg.title;

```

이런식으로 `find` 나 `filter`를 이용해서 간단히 표현할 수 있을 듯 합니다!